README.md

```diff
@@ -25,46 +25,68 @@
+* [Learn Concepts and Commands](#learn-concepts-and-commands)
+* [OpenWhisk Community and Support](#openwhisk-community-and-support)
+* [Project Repository Structure](#project-repository-structure)
 
+### Quick Start
 
-* [Quick Start](#quick-start) (Docker-Compose)
-* [Native development](#native-development) (Mac and Ubuntu)
-* [Kubernetes](#kubernetes-setup)
-* [Learn concepts and commands](#learn-concepts-and-commands)
-* [Issues](#issues)
-* [Slack](#slack)
+The easiest way to start using OpenWhisk is to install the "Standalone OpenWhisk". This is a full-featured OpenWhisk stack running as a Java
+process for convenience. Serverless functions run within Docker containers.
+- Pre-requesites:
+  - [Docker](https://docs.docker.com/install)
+  - [Java](https://java.com/en/download/help/download_options.xml)
+  - [Node.js](https://nodejs.org)
 
-### Quick Start
-The easiest way to start using OpenWhisk is to get Docker installed on Mac, Windows or Linux. The [Docker website](https://docs.docker.com/install/) has detailed instructions on getting the tools installed. This does not give you a production deployment but gives you enough of the pieces to start writing functions and seeing them run.
+```
+git clone https://github.com/apache/openwhisk.git
+cd openwhisk
+./gradlew core:standalone:bootRun
+```
 
+- When the OpenWhisk stack is up, it will open your browser to a functions Playground,
+typically served from http://localhost:3232. The Playground allows you create and run functions directly from your browser.
+
+- To make use of all OpenWhisk features, you will need the OpenWhisk command line tool called
+`wsk` which you can download from https://s.apache.org/openwhisk-cli-download.
+Please refer to the [CLI configuration](./docs/cli.md) for additional details. Typically you
+configure the CLI for the Standalone Controller as follows:
 ```
-git clone https://github.com/apache/openwhisk-devtools.git
-cd openwhisk-devtools/docker-compose
-make quick-start
+wsk property set \
+  --apihost 'http://localhost:3233' \
+  --auth '23bc46b1-71f6-4ed5-8c54-816aa4f8c502:123zO3xZCLrMN6v2BKK1dXYFpXlPkccOFqm12CdAsMgRU4VrNZ9lyGVCGuMDGIwP'
 ```
 
-For more detailed instructions or if you encounter problems see the [OpenWhisk-dev tools](https://github.com/apache/openwhisk-devtools/blob/master/docker-compose/README.md) project.
+- The Standalone Controller can be configured to deploy additional capabilities when that is desirable.
+Additional resources are available [here](./core/standalone/README.md).
+
+### Deploy to Kubernetes
 
-### Kubernetes Setup
+OpenWhisk can also be installed on a Kubernetes cluster. You can use
+a managed Kubernetes cluster provisioned from a public cloud provider
+(e.g., AKS, EKS, IKS, GKE), or cluster you manage yourself.
+Additionally for local development, OpenWhisk is compatible with Minikube,
+and Kubernetes for Mac using the support built into Docker 18.06 (or higher).
```

Comment:
@dgrove-oss can you take another look at this paragraph?

Reply:
Looks good.  Agree it makes more sense to lead with a managed cluster; that's the easy path for many users.

Reply:
thanks.